subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -94,9 +94,16 @@ In order to disable auto-detection, you can use the `org.gradle.java.installatio
 [[sec:provisioning]]
 == Auto Provisioning
 
-If no toolchain is found matching the requirements of the build, Gradle will automatically download a matching JDK from AdoptOpenJDK. By default, it will request a HotSpot JDK matching the current operating system and architecture.
+If no toolchain is found matching the requirements of the build, Gradle will automatically download a matching JDK from AdoptOpenJDK.
+By default, it will request a HotSpot JDK matching the current operating system and architecture.
```

Comment:
Can you change the default for the automatic downloads? If not we can simply say "If no toolchain is found matching the requirements of the build, Gradle will automatically download a matching JDK from AdoptOpenJDK for the current operating system and architecture."

Reply:
You can change the root server, though it needs to expose JDKs according to the AdoptOpenJDK ReST API. And you can also disable downloading.

Reply:
> If no toolchain is found matching the requirements of the build, Gradle will automatically download a matching JDK from AdoptOpenJDK.

:o: Suggestion:

> If Gradle can't find a locally available toolchain which matches the requirements of the build, it will automatically try to download it from AdoptOpenJDK.

And say something about how to disable this automatic download.

Reply:
The info about disabling auto download is couple lines below, with a section title to make it easily visible.